README.md

```diff
@@ -64,78 +98,98 @@ That's where this package comes in - it lets you write your queries with [ES2015
 
-### Babel preprocessing
-
-GraphQL queries can be compiled at build time using [babel-plugin-graphql-tag](https://github.com/gajus/babel-plugin-graphql-tag). Pre-compiling queries decreases the script initialization time and reduces the bundle size by potentially removing the need for `graphql-tag` at runtime.
 
-#### TypeScript
-Try this custom transformer to pre-compile your GraphQL queries in TypeScript: [ts-transform-graphql-tag](https://github.com/firede/ts-transform-graphql-tag).
+### Importing graphQL files
 
-#### React Native, Next.js
+_To add support for importing `.graphql`/`.gql` files, see [Webpack loading and preprocessing](#webpack-loading-and-preprocessing) below._
 
-Additionally, in certain situations, preprocessing queries via the webpack loader is not possible. [babel-plugin-import-graphql](https://www.npmjs.com/package/babel-plugin-import-graphql) will allow one to import graphql files directly into your JavaScript by preprocessing GraphQL queries into ASTs at compile-time.
+Given a file `MyQuery.graphql`
 
-E.g.:
-```javascript
-import myImportedQuery from './productsQuery.graphql'
-
-class ProductsPage extends React.Component {
+```graphql
+query MyQuery {
   ...
 }
 ```
 
-#### Create-React-App
+If you have configured [the webpack graphql-tag/loader](#webpack-loading-and-preprocessing), you can import modules containing graphQL queries. The imported value will be the pre-built AST.
 
-`create-react-app@2.0.0` does support the ability to preprocess queries using [evenchange4/graphql.macro](https://github.com/evenchange4/graphql.macro).
+```graphql
+import MyQuery from 'query.graphql'
+```
 
-If you're using an older version of `create-react-app`, check out [react-app-rewire-inline-import-graphql-ast](https://www.npmjs.com/package/react-app-rewire-inline-import-graphql-ast) to preprocess queries without needing to eject.
+#### Importing queries by name
 
-### Webpack preprocessing with `graphql-tag/loader`
+You can also import query and fragment documents by name.
 
-This package also includes a [webpack loader](https://webpack.js.org/concepts/loaders). There are many benefits over this approach, which saves GraphQL ASTs processing time on client-side and enable queries to be separated from script over `.graphql` files.
+```graphql
+query MyQuery1 {
+  ...
+}
 
-```js
-loaders: [
-  {
-    test: /\.(graphql|gql)$/,
-    exclude: /node_modules/,
-    loader: 'graphql-tag/loader'
-  }
-]
+query MyQuery2 {
+  ...
+}
 ```
 
-then:
+And in your JavaScript:
+```javascript
+import { MyQuery1, MyQuery2 } from 'query.graphql'
+```
 
-```js
-import query from './query.graphql';
+### Preprocessing queries and fragments
 
-console.log(query);
-// {
-//   "kind": "Document",
-// ...
-```
+Preprocessing graphQL queries and fragments to ASTs at built time can greatly increase load times.
```

Comment:
"at build time can greatly improve load times"

(I think load times will be less due to preprocessing, right?)

Reply:
Yep! Fixed